src/com/trollworks/gcs/character/GURPSCharacter.java

```diff
@@ -2724,7 +2724,7 @@ public int getIntegerBonusFor(String id, StringBuilder toolTip) {
      * @param specializationQualifier The specialization qualifier.
      * @return The bonus.
      */
-    public int getSkillComparedIntegerBonusFor(String id, String nameQualifier, String specializationQualifier) {
+    public int getSkillComparedIntegerBonusFor(String id, String nameQualifier, String specializationQualifier, String categoryQualifier) {
         return getSkillComparedIntegerBonusFor(id, nameQualifier, specializationQualifier, null);
```

Comment:
This just calls itself, in an infinite loop

Reply:
Duh.... sorry, was trying to cull out just these changed from my other instance of Eclipse.   It has been fixed in the latest commit.